Solita.LocalizationEditor.UI/Models/CategoryList.cs

```diff
@@ -47,6 +48,36 @@ public void AddTranslation(string languageId, string value)
             {
                 Translations.Add(languageId, value);
             }
+
+            public override bool Equals(object obj)
+            {
+                var translation = obj as Translation;
+                if (translation == null)
+                {
+                    return false;
+                }
+                if (
+                    string.Compare(translation.Key, Key) != 0 || 

```

Comment:
Just wondering why you didn't use string.Equals here since that seems more appropriate